my-react-app/src/app/screens/Login/fieldNames.js

```diff
@@ -0,0 +1,3 @@
+export const EMAIL_FIELD = 'email';
```

Comment:
Make a single `fields.js` file. And make an FORM_FIELDS object for this strings.

`export const FORM_FIELDS  = {EMAIL_FIELD : 'email',PASSWORD_FIELD : 'password'}`